### ISSUE TITLE
2 items + metadata  in Rose Early, Grass Scarce exhibit are not found when clicking the link for them in the "browse all" category

### ISSUE THREAD

**Issue by kelea99** [pulibrary/dpul#923]
there are 2 items in an unpublished DPUL exhibit that are not pulling in the metadata and the link goes to an item that doesn’t exist. I’ve reindexed the items in DPUL, looked to ensure they were tagged and view appropriately in Figgy (and the online cat/FA), which they do.  What am i  missing?
item 01 FA ; item 01 figgy ; item 01 DPUL (unpublished)
item 02 (Online Cat) ; item 02 figgy ;  item 02 DPUL (unpublished)

@hackartisan did find both items, displaying via these links:
item 01:  https://dpul.princeton.edu/rose-early_grass-scarce/catalog/hh63t0839
item 02 https://dpul.princeton.edu/rose-early_grass-scarce/catalog/8p58pd68c (

They also do not have titles in the items listing in the dashboard facet.  
@hackartisan also found that the search bar is not displaying for the exhibit either. @kelea99 is about to check the dashboard to see what is up. 

![Funky items](https://user-images.githubusercontent.com/45043014/121039251-ef540600-c77e-11eb-80a2-5d6f1ea825de.png)

**Comment by kelea99** [comment]
update: the search box was deactivated. I turned everything back on. it works fine.

**Comment by hackartisan** [comment]
When I index this collection locally everything comes through as expected.

When I look at the collection on the box, there are 20 resources instead of 16 in the database.

**Comment by hackartisan** [comment]
The numbers listed on those items are coming from the solr document id of the sidecars on those resources: 
```
> e.resources.select{ |r| r.data["noid"].nil?}.each { |r| puts r.solr_document_sidecars.map(&:document_id)}
2d93e97538bf224de4ba2167a6e14de1
55d17bd499d9af419eaedf60378a173f
```

**Comment by hackartisan** [comment]
Found a distinction between the 4 bad items and the others:

```
irb(main):026:0> e.resources.each { |r| puts "#{r.data['noid']} - #{r.url}"};nil
j6731452g - https://figgy.princeton.edu/concern/scanned_resources/91279e26-bfc4-4e1f-8384-41595781fb58/manifest?auth_token=[redacted]
3197xm463 - https://figgy.princeton.edu/concern/scanned_resources/2e864e25-1b0f-4db9-8865-7f55ba493ccf/manifest?auth_token=[redacted]
wh246s885 - https://figgy.princeton.edu/concern/scanned_resources/84ccca3f-cf2e-4d01-b8ef-fc9d76dcb73a/manifest?auth_token=[redacted]
7p88ch283 - https://figgy.princeton.edu/concern/scanned_resources/72c78c38-5432-418c-92c9-c6dc9574aebf/manifest?auth_token=[redacted]
2j62s4949 - https://figgy.princeton.edu/concern/scanned_resources/93e6376c-91a1-4ad0-bed5-0bce469e372b/manifest?auth_token=[redacted]
sq87bv364 - https://figgy.princeton.edu/concern/scanned_resources/009bbeb6-3951-4311-aad4-401c9a96bce6/manifest?auth_token=[redacted]
4x51hq50f - https://figgy.princeton.edu/concern/scanned_resources/f7f6ecd2-3bc1-4ca1-9df1-e91ec8d2a2db/manifest?auth_token=[redacted]
pk02ch200 - https://figgy.princeton.edu/concern/scanned_resources/fc345562-0895-4321-b8a3-ffc81ef8e27d/manifest?auth_token=[redacted]
6q182r60c - https://figgy.princeton.edu/concern/scanned_resources/4b73d943-b005-4c02-ae96-19060ad8b43c/manifest?auth_token=[redacted]
5712mb50v - https://figgy.princeton.edu/concern/scanned_resources/2784a5f4-3534-44b3-a08d-2690288bdda9/manifest?auth_token=[redacted]
4x51hj765 - https://figgy.princeton.edu/concern/scanned_resources/b9700fb8-1b1b-4c31-91a7-780d437a93cb/manifest?auth_token=[redacted]
41687n57h - https://figgy.princeton.edu/concern/scanned_resources/376423e3-9543-4aca-9619-ccd017503953/manifest?auth_token=[redacted]
qr46r491g - https://figgy.princeton.edu/concern/scanned_resources/0f4c10c3-6b1f-425e-ba15-1fb1ba24e7d4/manifest?auth_token=[redacted]
hh63t0839 - https://figgy.princeton.edu/concern/scanned_resources/e7ff5fe5-1938-467b-8d5a-3818858cec05/manifest?auth_token=[redacted]
vh53x083j - https://figgy.princeton.edu/concern/scanned_resources/42258884-f6fe-4cbd-8902-8dfce2140c09/manifest?auth_token=[redacted]
 - https://figgy.princeton.edu/concern/scanned_resources/376423e3-9543-4aca-9619-ccd017503953/manifest?manifest=https://figgy.princeton.edu/concern/scanned_resources/376423e3-9543-4aca-9619-ccd017503953/manifest
 - https://figgy.princeton.edu/concern/scanned_resources/376423e3-9543-4aca-9619-ccd017503953/manifest?manifest=https://figgy.princeton.edu/concern/scanned_resources/376423e3-9543-4aca-9619-ccd017503953/manifest
 - https://figgy.princeton.edu/concern/scanned_resources/376423e3-9543-4aca-9619-ccd017503953/manifest?manifest=https://figgy.princeton.edu/concern/scanned_resources/376423e3-9543-4aca-9619-ccd017503953/manifest
 - https://figgy.princeton.edu/concern/scanned_resources/e055d628-bf67-4dfa-ad1c-3c80e66cb6c2/manifest?manifest=https://figgy.princeton.edu/concern/scanned_resources/e055d628-bf67-4dfa-ad1c-3c80e66cb6c2/manifest
8p58pd68c - https://figgy.princeton.edu/concern/scanned_resources/e055d628-bf67-4dfa-ad1c-3c80e66cb6c2/manifest?auth_token=[redacted]
```

**Comment by hackartisan** [comment]
I deleted the 4 items with null noids from the db. Then I had to delete the 2 in question from the solr index manually. Once I deleted them the preferred index entries were able to display as desired.

It's not clear what caused the data to be this way, but the manifest urls resemble those for objects belonging to multivolume works. Perhaps they were initially part of a multivolume work? Something to keep an eye out for. In the meantime this appears resolved.
